ui/src/pages/program/components/tabulka/Přihlašovátko.tsx

```diff
@@ -15,11 +15,41 @@ type FormTlačítkoTyp =
   | "prihlasSledujiciho"
   | "odhlasSledujiciho";
 
+
+interface PotvrzeniModalProps {
+  formRef: MutableRef<HTMLFormElement | null>;
+  potvrzovatkoRef: MutableRef<any>;
+  aktivitaId: number;
+}
+
+const PotvrzeniModal: FunctionComponent<PotvrzeniModalProps> = ({
+  formRef,
+  potvrzovatkoRef,
+    aktivitaId,
+}) => {
+  const aktivita = useAktivita(aktivitaId);
+  return (
+    <div className="potvrzeniModalObal" ref={potvrzovatkoRef} onClick={(_) => {
+      potvrzovatkoRef.current.style.display = "none";
+    }}>
+      <div className="potvrzeniModal">
+        <h3>Opravdu se chceš odhlásit z aktivity{" " + aktivita?.nazev}?</h3>
+        <a href="#" onClick={(e) => {
```

Comment:
Funguje to dobře, díky.

Jen tě poprosím o explicitní možnost modal zavřít - kliknutí vedle není dost intuitivní.

Buďto křížek, nebo tlačítko Zavřít, nechám na tobě.